go.sum

```diff
@@ -609,6 +609,10 @@ github.com/lazyledger/merkletree v0.0.0-20201214195110-6901c4c3c75f h1:jbyPAH6o6
 github.com/lazyledger/merkletree v0.0.0-20201214195110-6901c4c3c75f/go.mod h1:10PA0NlnYtB8HrtwIDQAyTKWp8TEZ0zBZCGlYC/7+QE=
 github.com/lazyledger/nmt v0.2.0 h1:jKpC+XMSc4P1asLvutSwyMlgUDnAvv1HAVyzal21MbA=
 github.com/lazyledger/nmt v0.2.0/go.mod h1:tY7ypPX26Sbkt6F8EbPl3AT33B5N0BJe4OVPbq849YI=
+github.com/lazyledger/nmt v0.2.1 h1:/gSzFilF/fzykHiDDkwvnoyRdtntosgXP5feJbcTs1o=
+github.com/lazyledger/nmt v0.2.1/go.mod h1:tY7ypPX26Sbkt6F8EbPl3AT33B5N0BJe4OVPbq849YI=
+github.com/lazyledger/nmt v0.3.0 h1:MeWUG8nObGU45ZNrnaq+iqok2Mw2gVF+a4ldKfTGLRE=
+github.com/lazyledger/nmt v0.3.0/go.mod h1:tY7ypPX26Sbkt6F8EbPl3AT33B5N0BJe4OVPbq849YI=
```

Comment:
Surprising that both versions are used here 🤔 Maybe updating to `v0.3.0` can happen in a separate PR even.

Reply:
Yeah, that's an artifact from an attempt to update to `v0.3.0`. I tried but realize that `Push` semantics were changed and downgraded back, as that out of the scope.

Reply:
Is it necessary to remove the artifact? It should not hurt if merged.

Reply:
Just run ` go mod tidy -v` and commit the changes.

Reply:
And it's probably good to update consistently to `v0.2.1`. Also in `p2p/ipld/plugin/go.mod`